pinning.go

```diff
@@ -547,8 +555,11 @@ func (s *Server) handleListPins(e echo.Context, u *User) error {
 		}
 	}
 
+	if len(out) == 0 {
```

Comment:
is this really needed?

Reply:
Same as above. Btw context on this: if `out` didn't have any entries it'd return `nil` and that was messing up other tools that expected an empty list instead of `nil`

Reply:
ok